opm/parser/eclipse/EclipseState/Schedule/Connection.hpp

```diff
@@ -58,6 +59,7 @@ namespace Opm {
         double getSkinFactor() const;
         bool attachedToSegment() const;
         const Value<double>& getConnectionTransmissibilityFactorAsValueObject() const;
+        const Value<double>& getEffectiveKhAsValueObject() const;
```

Comment:
I am responsible for this `Value<T>` weirdness - it does serve a purpose but it really feels quite klunky. I appreciate you following the local style, but feel free to use a different approach than this `Value<T>` .